src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -241,8 +241,11 @@ export default class DiscussionArea {
 
     const currentTranscriptLine =
         this.#transcriptSeeker.currentTranscriptLine();
-    const currentTranscriptLineId =
-        currentTranscriptLine.transcriptLine.transcriptKey.id;
+    let currentTranscriptLineId;
+    if (currentTranscriptLine) {
```

Comment:
I am assuming currentTranscriptLine is not a boolean. If it is not a boolean then you should compare it to something. I am assuming you are checking if it is null or not.

Reply:
gotcha, updated!